localization/ndt_scan_matcher/config/ndt_scan_matcher.param.yaml

```diff
@@ -43,6 +43,11 @@
     # The number of particles to estimate initial pose
     initial_estimate_particles_num: 100
 
+    # The number of initial random trials in the TPE (Tree-Structured Parzen Estimator).
+    # This value should be equal to or less than 'initial_estimate_particles_num' and more than 0.
+    # If it is equal to 'initial_estimate_particles_num', the search will be the same as a full random search.
+    n_startup_trials: 30
```

Comment:
Please update readme parameters as well

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/5078/commits/9887fcf41aa1ae2ed75d667dcb8f15df5afa35a9